app/src/main/res/layout/activity_quiz_result.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think you forgot to remove that layout file...

Reply:
Yes most probably